storm-core/src/ui/public/templates/index-page-template.html

```diff
@@ -229,6 +249,10 @@
       <td>{{uptime}}</td>
       <td>{{slotsTotal}}</td>
       <td>{{slotsUsed}}</td>
+      <td>{{totalMem}}</td>
+      <td>{{usedMem}}</td>
+      <td>{{totalCpu}}</td>
+      <td>{{usedCpu}}</td>
```

Comment:
Sure, adding it.

For non-RAS scheduler, the totalMem and totalCpu would be default(3000 and 400), and the usedMem and usedCpu will be always 0 and 0, no matter whether there is topology or not.